src/styles/blocks/logo.scss

```diff
@@ -0,0 +1,21 @@
+.logo {
+  display: flex;
```

Comment:
Add 'align-items: center;' to vertically center the logo image within the logo container.